content/en/docs/concepts/scheduling-eviction/api-eviction.md

```diff
@@ -0,0 +1,16 @@
+---
+title: API-initiated Eviction
+content_type: concept
+weight: 70
+---
+
+{{< glossary_definition term_id="api-eviction" length="short" >}} </br>
+
+You can request eviction by directly calling the Eviction API 
```

Comment:
Link to https://kubernetes.io/docs/tasks/administer-cluster/safely-drain-node/#eviction-api ?

Reply:
It is linked in the glossary definition (so looks OK in staging at: https://deploy-preview-27739--kubernetes-io-master-staging.netlify.app/docs/concepts/scheduling-eviction/api-eviction/)

Reply:
looks good.

Now, I think some of the contents of https://kubernetes.io/docs/tasks/administer-cluster/safely-drain-node/#eviction-api can be put here as well. Note that that doc is a "task", meaning that it has a tutorial structure. In "concepts", we should make it more descriptive, explaining the fields and responses. But this is definitely for a follow up.

Reply:
I'll create a new issue for this after we publish this doc